crates/runtime/src/lib.rs

```diff
@@ -291,6 +292,17 @@ impl_runtime_apis! {
         fn contract_class_by_class_hash(class_hash: ClassHashWrapper) -> Option<ContractClassWrapper> {
             Starknet::contract_class_by_class_hash(class_hash)
         }
+
+        /// Retrieves a transaction's receip from the storage based on the transaction hash.
+        fn get_transaction_by_hash(tx_hash: H256) -> Option<TransactionReceiptWrapper> {
+            // TODO: Naive approach.
```

Comment:
Can you confirm that the block only contains `Transactions` and not the receips. Am I able to find receips directly going into the storage, or for now, I can stick with transactions and I'll update this once receips are present in the block?

Reply:
Sorry, I just seen the other PR for transaction receipt. I'll focus on Transaction.